deploy/PowerSwitchFactory.ts

```diff
@@ -13,7 +13,8 @@ const deployFunc = async function ({
     from: deployer,
     args: [],
     log: true,
-    contract: "PowerSwitchFactory",
+    contract:
+      "src/contracts/powerSwitch/PowerSwitchFactory.sol:PowerSwitchFactory",
```

Comment:
why isthis necessary? is there more than one PowerSwitchFactory? What's the difference between them?

Reply:
The difference between the alchemistcoin/alchemist PowerSwitch and this PowerSwitch is that the latter receives an uint256 to update the status to online after a particular timestamp.